Add files via upload

MaxPosthumus opened the pull request:
I don't know whether to mimic bh-7-1.tex in the bh-problems directory or the files in the assignment directory. (The files in the assignment directory are not all in the same format so I chose to mimic bh-7-1.tex. ) In particular, I am unsure whether to include the header/trailer files and if I need \subsection{BH.1.7} or \begin{exercise}[BH.1.7].

ndvanforeest reviewed (changes requested):
Please put dollars around player names like $A$, $B$. Also, it's best to write $1$ ,etc, because in some font types, 1 in the text is different from $1$. If you like to improve your programming skills, write a regular expression to repair this problem in all files. 

End a question with a ?

I don't understand the very first question. What are "the original questions"? 

tie -> die

I guess that the words "that overall" is superfluous/confusing.

What is "value" vs "probability value"?

MaxPosthumus commented:
The "original questions" refers to the exercises in Blitzstein. Lingwei uses this reference in all the assignment exercises. In the main file of the PT assignments (which is not uploaded yet) this is shortly addressed. By the way I replaced the file by copy pasting from latex (where I made the adjustments), but I now realize that this makes it unclear what parts I have changed.
